tests/test_streams.rs

```diff
@@ -277,12 +288,18 @@ fn test_assorted_2() {
 
     // call xpending here...
     // this has a different reply from what the count variations return
-    let reply: StreamPendingReply = con.xpending("k99", "g99").unwrap();
-    assert_eq!(reply.count, 3);
-    assert_eq!(reply.start_id, "1000-1");
-    assert_eq!(reply.end_id, "1001-1");
-    assert_eq!(reply.consumers.len(), 1);
-    assert_eq!(reply.consumers[0].name, "c99");
+    let data_reply: StreamPendingReply = con.xpending("k99", "g99").unwrap();
+
+    assert_eq!(data_reply.count(), 3);
+
+    if let StreamPendingReply::Data(data) = data_reply {
+        assert_eq!(data.start_id, "1000-1");
+        assert_eq!(data.end_id, "1001-1");
+        assert_eq!(data.consumers.len(), 1);
+        assert_eq!(data.consumers[0].name, "c99");
+    } else {
+        panic!("Expected StreamPendingReply::Data but got Empty");
```

Comment:
I think the `assert_eq!(data_reply.count(), 3);` case above will catch this before it reaches this part, right? You can prob remove this else clause.